test/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/returns/ReturnsServiceImplSpec.scala

```diff
@@ -1875,8 +1875,9 @@ class ReturnsServiceImplSpec extends AnyWordSpec with Matchers with MockFactory
     "handling requests to list returns" must {
 
       val cgtReference       = sample[CgtReference]
+      val today: LocalDate   = LocalDate.now()
       val (fromDate, toDate) =
-        LocalDate.now().minusYears(config.numberOfTaxYearsForReturns).plusDays(1L) -> LocalDate.now()
+        today.minusYears(config.numberOfTaxYearsForReturns).plusDays(1L) -> TimeUtils.getTaxYearEndDateInclusive(today)
```

Comment:
This is getting silly. I know you didn't write this, but this tuple is unnecessary.

```suggestion
        val fromDate = today.minusYears(config.numberOfTaxYearsForReturns).plusDays(1L)
        val toDate = TimeUtils.getTaxYearEndDateInclusive(today)
```